### PR TITLE
V2 74 fix cors for osm map

### PR DESCRIPTION
**Pull request by RobertoMaurizzi**
Bump to 2.74 (the PR also includes 2.73, they're both simple fixes to the admin setup and 2.74 fixes a Django model warning introduced with 2.73, a nullable field that can't be nullable nut only blank-able)

### CODE REVIEW

File: simple_locations/admin.py [hunk 1]
```diff
@@ -27,11 +28,27 @@ class AreaTypeAdmin(admin.ModelAdmin):
...
-    list_display = ( 'name', 'kind', 'location', 'code')
+class AreaChildrenInline(admin.TabularInline):
+    model = Area
+    fields = ['name', 'code', 'kind']
+    show_change_link = True
+    extra = 0
+
+
+class AreaAdmin(MPTTModelAutocompleteAdmin, admin.OSMGeoAdmin):
+    # default_lon = getattr(settings, 'GIS_DEFAULT_LAT', -8.8742)
```
Comment (PeteCoward) [inline simple_locations/admin.py:39]: why this change?

**Review by PeteCoward — COMMENTED** [review]
Can you explain what actually fixes the CORS issue?

File: simple_locations/admin.py (hunk 1 above)
Comment (RobertoMaurizzi) [reply simple_locations/admin.py:39]: Because I'd like to see a more detailed map if I ever go on to add coordinates from our geojson files/tempu.tl database ^^;

**Comment by RobertoMaurizzi** [comment]
The CORS-caused "broken map tiles" problem happens because OpenLayers will request tiles from `openstreetmap.org` without defining any CORS behavior. The overridden bit of javascript that calls `OpenLayers.Layer.OSM("OpenStreetMap (Mapnik)", undefined, {tileOptions: {crossOriginKeyword: null}})` instead of `OpenLayers.Layer.OSM("OpenStreetMap (Mapnik)")` disables CORS headers/checks for this request.

Now, since I didn't notice we had django-cors-header set up (in LI) I'm not really sure if this fix will be enough for a server instance (at least without adding openstreetmap.org to the valid CORS hosts). This change however will allow the local `runserver` to display the tiles on Chrome (Safari was working, apparently it doesn't care much about CORS for sites running from localhost... 🤔 )
